configure-esp32

```diff
@@ -4,4 +4,7 @@ set -e
 
 cd /opt/nodemcu-firmware
 
+# Basically revert windows lineends. Only needed if checkout is done under windows.
+git submodule foreach --recursive git checkout .
```

Comment:
Can I suggest to wrap a condition around this? No need to burn extra CPU cycles on *nix platforms.

Reply:
Sure would like to but after a quick search I could not dig up anything that would tell me what host system I am on.
@marcelstoer do you have a hint here?

Reply:
I could guess from the mountpoint of the firmware
Unter Windows it looks like ```//10.0.75.1/C on /opt/nodemcu-firmware type cifs```
what does it lock like on linux/Mac?